### DISCUSSION TITLE
How to move the story menu week select up? (source)

### DISCUSSION THREAD

**Discussion by ElitePhoenix** [FunkinCrew/Funkin#2006] [Q&A]
Im trying to move the week select upwards a bit and make the selected week at the top of the screen, how do i move it?
![MoveSelectedWeek](https://user-images.githubusercontent.com/87148958/173208793-8438ff89-4846-4e16-820c-8d8ca6c4045f.png)

**Comment by Snirozu** [comment]
change the `y` of `weekThing`

**Reply by ElitePhoenix** [reply]
it moved back to the same place for some reason and moved the difficulty select up instead

**Reply by Snirozu** [reply]
you need to set the y of MenuItem in update() function
for example
``` haxe
y = FlxMath.lerp(y, 0, 0.17);
```
